runtime/mainnet/src/lib.rs

```diff
@@ -317,6 +323,7 @@ impl frame_system::Config for Runtime {
 	type SS58Prefix = SS58Prefix;
 	/// The action to take on a Runtime Upgrade
 	type OnSetCode = cumulus_pallet_parachain_system::ParachainSetCode<Self>;
+	/// The maximum number of consumers allowed on a single account.
 	type MaxConsumers = frame_support::traits::ConstU32<16>;
```

Comment:
In practice this is the number of assets an account can hold without additional deposit. Maybe take the chains off and make it like 256?

Reply:
yeah, i mean with the philosophy of this chain, we should set it to u32 max.